app/Http/Controllers/AdminDash.php

```diff
@@ -469,8 +469,17 @@ public function disallowVisitReq($id) {
     }
 
     public function allowVisitReq(Request $request) {
+        $validator = $request->validate([
+            'cid' => 'required'
+        ]);
+
         $id = $request->cid;
         $visitrej = VisitRej::where('cid', $id)->first();
+
```

Comment:
@VioletSnow08 please eliminate one of these blank lines - there should just be 1

Reply:
Fixed!